concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-78	78	colourful	COLORFUL	2997	1	4.80952380952381	0.238095238095238	0.19047619047619	0.428571428571429	0.333333333333333	0.87287156094397	0.700340053457026	0.511766315719159	1.02817452659695	0.912870929175277	Visual	0.76984126984127	11	2.484906649788	9
+Lynott-2009-423-79	79	compact			1	4.76190476190476	2.9047619047619	0.476190476190476	0.19047619047619	0.285714285714286	0.538958431120796	1.72929933512859	1.12334534400814	0.601585207518238	0.717137165600636	Visual	0.530386740331492	17	2.89037175789616	7
+Lynott-2009-423-80	80	conical			0.619047619047619	4.69230769230769	4.07692307692308	0.153846153846154	0	0.076923076923077	0.630425171956115	1.32045058354705	0.375533808099405	0	0.277350098112615	Visual	0.521367521367521	2	1.09861228866811	7
+Lynott-2009-423-81	81	contoured			0.952380952380952	4.15	3.6	0.35	0.1	0.05	0.812727700887249	1.46539019413009	0.93330200448673	0.307793505625546	0.223606797749979	Visual	0.496969696969697	0	0	9
+Lynott-2009-423-82	82	cooing			0.904761904761905	2.15789473684211	0.105263157894737	4.84210526315789	0	0	1.34425352663091	0.458831467741124	0.50145985712128	0	0	Auditory	0.681481481481481	0	0	6
+Lynott-2009-423-83	83	cool			1	2.42857142857143	4.33333333333333	0.380952380952381	0.285714285714286	1.85714285714286	1.36277028773849	0.856348838577676	0.804747816162957	0.560611910581388	1.59014824106793	Haptic	0.435897435897436	40	3.71357206670431	4
+Lynott-2009-423-84	84	crackling			1	2.80952380952381	1.38095238095238	4.28571428571429	0.904761904761905	1.33333333333333	1.36451631060415	1.28359613882991	0.956182887467515	0.995226703056239	1.52752523165195	Auditory	0.315555555555556	2	1.09861228866811	9
+Lynott-2009-423-85	85	craggy			0.714285714285714	3.46666666666667	2.66666666666667	1	0.133333333333333	0.133333333333333	1.35576371027375	1.83873662631503	1.25356634105602	0.351865775274498	0.516397779494322	Visual	0.45045045045045	1	0.693147180559945	6
+Lynott-2009-423-86	86	crashing			1	4.61904761904762	1.23809523809524	4.57142857142857	0.523809523809524	0.142857142857143	0.49761335152812	1.54611648670991	0.74642002729218	1.03048763306736	0.654653670707977	Visual	0.40343347639485	5	1.79175946922806	8
+Lynott-2009-423-87	87	creaking			1	2.04761904761905	1.14285714285714	4.47619047619048	0	0	1.46547570693582	1.31475147026783	0.813575295780767	0	0	Auditory	0.583850931677019	2	1.09861228866811	8
+Lynott-2009-423-88	88	creamy			1	3.80952380952381	2.76190476190476	0.19047619047619	2.38095238095238	4.47619047619048	1.5039630187956	1.75797502555531	0.511766315719159	1.85677652064513	1.20909253653505	Gustatory	0.314685314685315	4	1.6094379124341	6
+Lynott-2009-423-89	89	creased	WRINKLED	380	0.952380952380952	4.7	3.65	0.15	0.15	0.15	0.656946685331786	1.22581873821025	0.489360484929593	0.670820393249937	0.670820393249937	Visual	0.517045454545454	2	1.09861228866811	7
```

Comment:
Not wrong... but WRINKLED currently seems to both refer to wrinkled skin and wrinkled clothes. Not sure whether that shouldn't be split up at some point.

Reply:
I think the property of being wrinkled is similar in both cases and _creased_ can also refer to both. Therefore, I would not split it up.

Reply:
Yes, not at this point. Only if we find clear cases that distinguish it.